airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -410,6 +414,29 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
     quoteString = (quoteString == null ? database.getMetaData().getIdentifierQuoteString() : quoteString);
     database.setSourceConfig(sourceConfig);
     database.setDatabaseConfig(jdbcConfig);
+    LOGGER.info("Data source product recognized as {}:{}",
+        database.getMetaData().getDatabaseProductName(),
+        database.getMetaData().getDatabaseProductVersion());
+    final ResultSet tables = database.getMetaData().getTables(null,
+        null,
+        "%",
+        new String[] {JDBC_TABLE});
+    while (tables.next()) {
+      final String tableName = tables.getString(JDBC_COLUMN_TABLE_NAME);
+      final ResultSet indexInfo = database.getMetaData().getIndexInfo(null,
+          null,
+          tableName,
+          false,
+          false);
+      LOGGER.info("Discovering indexes for table \"{}\"", tableName);
+      while (indexInfo.next()) {
+        LOGGER.info("Index name: {}, Column: {}, Unique: {}",
```

Comment:
This should be a info or debug?

Reply:
Saying because there is some suggestion from community to remove some excessive info logs from databases https://github.com/airbytehq/airbyte/pull/24851

Reply:
No strong feelings for either for me. More than happy to upgrade it to `Debug` for this information to stand out during the logs. Thoughts? @rodireich, @akashkulk?

Reply:
My thinking is that if this log is not overly verbose and can help tackling customer cases - especially cloud where turning debug logging on is not trivial, then it should be `INFO`

Reply:
+1 on that. This is not too verbose (1 log per stream + 1 overall) and will greatly help in debugging

Reply:
In big Schemas this becomes very big, i have cases with more than 3000 lines so it completely blocks the log window.

Reply:
Fair point. Let's do this : 
1. Log Db version once here in `createDatabase`
2. Move logging indices to the `read()` version, since we have access to the configured catalog there. Probably close to where we are  logging schema change/validating cursor fields. 

Fine to do in a separate PR as well.